examples/tests/scala/org/pantsbuild/example/specs2/BUILD

```diff
@@ -0,0 +1,9 @@
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+junit_tests(
+  dependencies=[
+    '3rdparty/jvm/org/specs2:specs2-junit_2.11',
+    '3rdparty:junit',
```

Comment:
I thought the junit jar dep was auto-injected by the junit_tests target, making this dep un-needed / redundant.